### DISCUSSION TITLE
How do i get the thumbnail without black/empty section up and bottom

### DISCUSSION THREAD

**Discussion by ExyXyz** [JuanBindez/pytubefix#270] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/3d909fe6-5638-4b5e-b075-94cf01c92fb3)
https://i.ytimg.com/vi/whQ9VcK0pIA/sddefault.jpg
it appear like there was a black bar/empty section up and bottom how do i get the thumbnail without that

**Comment by felipeucelli** [comment, marked as answer]
pytubefix just returns the url that YouTube provides.

Some clients, maybe for compatibility reasons, have this border

You can try changing the client to `MWEB` or any other WEB based client.

```python

url = 'URL'

yt = YouTube(url, 'MWEB')

print(yt.thumbnail_url)
```

**Reply by ExyXyz** [reply]
so i tested it and its working thankyou
